src/features/headerFeature/views/components/styleMode/StyleMode.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function StyleMode() {
+    return (
+        <div>
+            
+        </div>
+    )
+}
+
+export default StyleMode
```

Comment:
If this component is mode button, you will need to change name. For example ThemeSwitcher, or ThemeSwitchRadio, or else.